includes/admin/give-metabox-functions.php

```diff
@@ -645,7 +645,7 @@ function give_docs_link( $field ) {
 	$field['url']         = isset( $field['url'] ) ? $field['url'] : 'https://givewp.com/documentation';
 	$field['title']         = isset( $field['title'] ) ? $field['title'] : 'Documentation';
```

Comment:
At first I thought we'd be fine since the string is escaped in the field function, but then it wouldn't be properly internationalized. Updating now. Thanks!